### DISCUSSION TITLE
Week 3 Thinking Cap

### DISCUSSION THREAD

**Discussion by skarakas1** [albertkun/21S-ASIAAM-191A#95] [Past Thinking Caps]
Open source shares many of the same problems that face democracy generally. Individuals and groups can exert undue influence, and misinformation can be weaponized. Although there are issues with open source, it still provides a path forward via its democratic nature.
The best argument for open source may be looking at the alternatives. Closed source tech providers will take every possible advantage to maximize profit and monopolize, as we have seen with companies like Google, Facebook, and Amazon. These companies profit from user data, often acquired unbeknownst to the users, or in less-than-transparent ways, and then attempt to dodge responsibility when data breaches happen (i.e. they are not able to effectively protect the data that they get from users and capitalize on).
After reviewing ethicalsource.dev, I think that these types of ethical open source licenses are definitely sensible moving forward. To some extent, they may protect open source software from being taken advantage of by large tech companies (if for instance those companies can be shown to be complicit in international human rights law violations).
Science fiction author Ted Chiang (speaking specifically about AI) has stated that “most of our fears about technology are best understood as fears or anxiety about how capitalism will use technology against us. And technology and capitalism have been so closely intertwined that it’s hard to distinguish the two.” (The Ezra Klein Show, https://www.nytimes.com/2021/03/30/podcasts/ezra-klein-podcast-ted-chiang-transcript.html)
I think that capitalism is grasping for straws in need of new frontiers of exploitation, and the internet offers potentially huge gains for minimal investments. NFTs and cryptocurrencies are a great example of this: their value is (seemingly) entirely based on perception and speculation. Large businesses and individual investors alike see this as a potential windfall because so many tech stocks have seen exponential growth. But the stock market is based on perceived value. Tesla, for instance, only posted net income in 2020, and even that was not based on their manufacturing but on their collection of regulatory credits (https://www.cnn.com/2021/01/31/investing/tesla-profitability/index.html), yet share prices are soaring.
Blockchain technologies, while currently trending, may prove to be unsustainable due to the incredible amounts of computing power necessary to ‘mint’ tokens and coins (https://www.nytimes.com/2021/04/14/climate/coinbase-cryptocurrency-energy.html). One Ethereum transaction is estimated to have the same carbon footprint of an EU resident’s power consumption for 4 days. That is around 20 kilograms of CO2, compared to 36 grams of CO2 for 36 hours of Netflix streaming (https://memoakten.medium.com/the-unreasonable-ecological-cost-of-cryptoart-2221d3eb2053). The cost of a single-edition NFT is equivalent to 1 month’s electricity consumption, or boiling a tea kettle 4.5 thousand times. A few clicks can be essentially equivalent to flying a jet for 2 hours.
I think that Chiang’s argument is compelling, and a call for action against speculation and neoliberal capitalism. Whatever downsides there are to open source I believe they are dwarfed by concerns about closed source. I am old enough to have grown up with dial-up internet and to have been an early user of Napster (!!!). My understanding is that the internet was largely created to democratize information, and I hope that that trajectory prevails. If I had to choose between anarchy and oligarchy, I would choose anarchy every time.

**Comment by rachan2023** [comment]
Closed source tools are a reflection on capitalism and it is unfortunate that corporations try to take as much data as they possibly can once users sign the mandatory user agreements. Since corporations are not transparent about how they are using or storing this data, there are many potential security risks, especially for users. A benefit of open source tools is that users do not have to worry about big corporations using their data without them knowing and people can use open source tools however they want, which is a double-edged sword in itself. People can use these tools for social change while others can use these tools to harm others. Open source tools are useful for facilitating collaboration and people with ill-intentions do make open source tools harmful for society. Additionally,  I do agree with Ted Chiang that it is hard to distinguish capitalism and technology since they are both so closely intertwined. This is particularly important now because a majority of websites and applications have a blatant capitalistic aspect, such as advertisements that encourage users to buy a product or subscription. I, too, hope that information continues to become democratized.
